packages/qwik/src/core/use/use-context.ts

```diff
@@ -134,6 +134,7 @@ export const createContextId = <STATE = unknown>(name: string): ContextId<STATE>
   assertTrue(/^[\w/.-]+$/.test(name), 'Context name must only contain A-Z,a-z,0-9, _', name);
   return /*#__PURE__*/ Object.freeze({
     id: fromCamelToKebabCase(name),
+    name, // keep the original name to be used when context is being used but not yet provided in one of the parents
```

Comment:
Can't we just use the id? I know it's not much, but it's still data that we keep just for the dev

Reply:
I was initially using it but displaying the original context id will make it more appropriate because displaying `user-profile` instead of `UserProfile` (the original context id) will be somehow a misleading, that's why i added the name to not make any confusion.